src/enkf/params.f90

```diff
@@ -243,6 +243,7 @@ module params
 integer(i_kind),public :: ntiles=6
 integer(i_kind),public :: nx_res=0,ny_res=0
 logical,public ::l_pres_add_saved
+logical,public ::l_fv3reg_filecombined =.true.  !=.true., the dynvar and tracer files  would be combined for enkf fv3_reg
```

Comment:
Good question.  I keep the default =.true., because it had been the case using the combined files till we add this new function. So,  default=.true would make the current system using combined files don't need change anything to run the new version of enkf_fv3reg.